### DISCUSSION TITLE
3rd-party Vue component is working but not visible

### DISCUSSION THREAD

**Discussion by tanshuai** [quasarframework/quasar#8971] [General - Components / Directives / etc]
I've tried to use 3rd-party Vue component https://github.com/sandanat/vue-pdf-app to display PDF content in Quasar App, the component is working via console log when the PDF file was loaded & parsed without errors, but not content can display on the screen. 

The standalone Vue app can work and display without problem, but it does not display in Quasar App.

Here is my code in Quasar:

```
<template>
  <q-page padding>
    <vue-pdf-app pdf="test.pdf">
    </vue-pdf-app>
  </q-page>
</template>

<script>
import VuePdfApp from 'vue-pdf-app'
import 'vue-pdf-app/dist/icons/main.css'

export default {
  name: 'PageName',
  components: {
    VuePdfApp
  }
}
</script>

```

**Comment by rstoenescu** [comment]
Did you add your test.pdf to the /public folder?

**Reply by tanshuai** [reply]
Yes, I did.  The console log outputs the test.pdf is loaded and prints the PDF document property information.

**Reply by lyle45** [reply]
Did you make sure it has proper height and width? Maybe give it a class
